app/assets/stylesheets/components/buttons.scss

```diff
@@ -460,6 +460,12 @@ label[class*='crayons-btn']:focus-within // label for input[type="file"] made to
     box-shadow: var(--focus-ring);
   }
 
+  &[aria-pressed="true"] {
+    --bg: var(--base-inverted);
```

Comment:
One thing I noticed is that in light mode we see a background on these elements, but in dark mode we don't:
![light mode](https://user-images.githubusercontent.com/20773163/144586053-77577220-77ed-4618-848a-e3c323cf3272.png)
![dark mode](https://user-images.githubusercontent.com/20773163/144586058-3d0eeaec-fc65-4fdf-a252-ef67dedae50e.png)

Reply:
It's weird... Because on light theme we use light grey as body background therefore white tab is clearly visible... 
On dark theme we use black as body background and the opposite color for selected tab (opposite for white is also black 🤷‍♂️) so these obviously do not differ visually. Lemme think what do about it, as I'm not sure myself actually :D

Reply:
I think I'll fix it after merging #15653 since that PR introduces better handling of color variables and it will just be easier to fix it then rather than now.